scrapper.py

```diff
@@ -1,6 +1,18 @@
 """
 Scrapper implementation
 """
+import json
+import re
+import os
+import requests
+from bs4 import BeautifulSoup
+
+from core_utils.article import Article
+from core_utils.pdf_utils import PDFRawFile
+
+from constants import ASSETS_PATH
+# from constants import CRAWLER_CONFIG_PATH ... idk what to do with this
```

Comment:
use CONFIG_PATH to get all scrapper parameters - seed urls, number of articles, headers etc